modules/core/src/test/java/org/apache/ignite/testframework/junits/GridAbstractTest.java

```diff
@@ -864,6 +867,111 @@ protected Ignite startGrid(String igniteInstanceName, IgniteConfiguration cfg, G
     }
 
     /**
+     * Starts new grid of given version and name <b>in separate JVM</b>.
+     *
+     * Uses an ignite-core artifact in the Maven local repository,
+     * if it isn't exists there, it will be downloaded and stored via Maven.
+     *
+     * @param igniteInstanceName Instance name.
+     * @param ver Ignite version.
+     * @param clos IgniteInClosure for post-configuration.
+     * @return Started grid.
+     * @throws Exception If failed.
+     */
+    protected Ignite startGrid(String igniteInstanceName, String ver,
+        IgniteInClosure<IgniteConfiguration> clos) throws Exception {
+        return startGrid(igniteInstanceName, ver, null, clos, null, true);
+    }
+
+    /**
+     * Starts new grid of given version and name <b>in separate JVM</b>.
+     *
+     * Uses an ignite-core artifact in the Maven local repository,
+     * if it isn't exists there, it will be downloaded and stored via Maven.
+     *
+     * @param igniteInstanceName Instance name.
+     * @param ver Ignite version.
+     * @param cfg Ignite configuration.
+     * @param clos IgniteInClosure for post-configuration.
+     * @return Started grid.
+     * @throws Exception If failed.
+     */
+    protected Ignite startGrid(String igniteInstanceName, String ver, IgniteConfiguration cfg,
+        IgniteInClosure<IgniteConfiguration> clos) throws Exception {
+        return startGrid(igniteInstanceName, ver, cfg, clos, null, true);
+    }
+
+    /**
+     * Starts new grid of given version and name <b>in separate JVM</b>.
+     *
+     * Uses an ignite-core artifact in the Maven local repository,
+     * if it isn't exists there, it will be downloaded and stored via Maven.
+     *
+     * @param igniteInstanceName Instance name.
+     * @param ver Ignite version.
+     * @param cfg Ignite configuration.
+     * @param clos IgniteInClosure for post-configuration.
+     * @param jvmArgs Additional JVM arguments.
+     * @param resetDiscovery Reset DiscoverySpi at the configuration.
+     * @return Started grid.
+     * @throws Exception If failed.
+     */
+    protected Ignite startGrid(String igniteInstanceName, final String ver, IgniteConfiguration cfg,
+        IgniteInClosure<IgniteConfiguration> clos, final Collection<String> jvmArgs,
+        boolean resetDiscovery) throws Exception {
+        assert isMultiJvm() : "MultiJvm mode must be switched on for the node stop properly.";
+
+        assert !isFirstGrid(igniteInstanceName) : "Please, start node of current version first.";
+
+        if (cfg == null)
+            cfg = optimize(getConfiguration(igniteInstanceName));
+
+        final String pathToArtifact = MavenUtils.getPathToIgniteCoreArtifact(ver);
```

Comment:
Is this variable necessarry?
It has to be declared closer to usage(first usage on line 955)